portal/www/portal/dashboard.php

```diff
@@ -37,49 +38,18 @@
 
 
 $user = geni_loadUser();
+
+// TODO: ask tom here!
```

Comment:
The issue Charlie is pointing out is the redirect loop caused by the redirection to home.php. The old (v2.33) home page punted by printing "Unable to load user record." if no user was loaded on the home page. We should do that or improve it by redirecting to the error page using that or something like it as the error message.